generateCalendarEvents.js

```diff
@@ -0,0 +1,65 @@
+const matter = require("gray-matter");
+const ics = require("ics");
+const communities = require("./communities");
+const fs = require("fs");
+
+
+// get all events
+communities.forEach(community => {
+  if (!fs.existsSync(`./static/${community.name}`)) {
+    fs.mkdirSync(`./static/${community.name}/`);
+  }
+
+  console.log(`Generating events for ${community.name}.`)
+  fs.readdir(`./content/${community.name}/sessions`, (err, files) => {
+    if (err) {
+      console.error("Could not list the directory.", err);
+      process.exit(1);
+    }
+
+    files.forEach((file, index) => {
+      const content = fs.readFileSync(`./content/${community.name}/sessions/${file}`)
+      const frontMatter = matter(content).data
+      if (new Date(frontMatter.dateTime) < new Date()) {
+        return console.log(`"${frontMatter.title}" has passed. Not creating a calendar event.`)
+      }
+
+      if (!frontMatter.dateTime) {
+        return console.log("Not creating event for template.")
+      }
+      const eventDate = new Date(frontMatter.dateTime)
+
+      const event = {
+        start: [
+          eventDate.getFullYear(),
+          eventDate.getMonth() + 1, // Months are zero indexed
+          eventDate.getDate(),
+          eventDate.getHours(),
+          eventDate.getMinutes()
+        ],
+        duration: getDuration(frontMatter.durationInMinutes),
+        title: frontMatter.title,
+        description: `A ${frontMatter.type} event.`,
+        location: "On Zoom - find link on the #skillsharing channel on Slack"
```

Comment:
this text is specific to the MozFest community. how will the script work with SU?

Reply:
I see the script goes per community and we are commenting out SU now anyway. 
no need to fix right now, but let's think about this for the future. I think this CTA could live in the communities.json config...

Reply:
Maybe this could be {frontmatter.location} with a fallback to a set string.

Reply:
yes, that seems like an elegant solution. 
could you make that adjustment before I merge?